index.html

```diff
@@ -328,6 +471,14 @@ <h5>NEWS</h5>
                                 <li><a href="https://www.nytimes.com/section/us"><U class="S">U.S.</U></a></li>
                                 <li><a href="">Politics</a></li>
                                 <li><a href="">Election 2020</a></li>
+                                <li><a href="https://www.nytimes.com/">Home Page</a></li>
+                                <li><a href="https://www.nytimes.com/section/world">World</a></li>
+                                <li><a href="https://www.nytimes.com/section/us"><U class="S">U.S.</U></a></li>
+                                <li><a href="">Politics</a></li>
+                                <li><a href="">Election 2020</a></li>
+                                <li><a href="https://www.nytimes.com/section/us"><U class="S">U.S.</U></a></li>
+                                <li><a href="">Politics</a></li>
+                                <li><a href="">Election 2020</a></li>
```

Comment:
There are a tags with empty href. Remember to use a hash(#) symbol in between the quotes, if there is no specific file path to include.